test/functional/Java11andUp/src/org/openj9/test/java_lang/Test_String.java

```diff
@@ -268,4 +268,105 @@ public void testLines() {
 			// Expected exception
 		}
 	}
+	
+	/*
+	 * Test Java 12 API String.indent
+	 */
+	// TODO: move to Java12andUp
```

Comment:
Please address the TODO

Reply:
Will move to Java12andUp once the #4105 test framework commit goes through

Reply:
@sharon-wang #4105 has been merged.  Can you rebase this and address the TODO?